Resources/ServerInfo/_CP14/Guidebook_EN/JobsTabs/Innkeepers.xml

```diff
@@ -0,0 +1,32 @@
+<Document>
+  # Innkeeping
+  The innkeeper is a role that primarily stays within the town. The responsibilities of an innkeeper include gardening, bartending, and cooking.
+
+  Within the inn, an innkeeper will have access to three main areas:
+
+  ## The Garden
+  The garden, or farm, is an important asset for any chef. A few seedbeds can be found here, along with a well and some basic supplies.
+  To begin farming, locate the supply table in the garden. Usually, a crate will contain a variety of seeds, while a table will hold farming tools like a bucket and hoe. Grab a seed variety and use it on the ground to plant it. Wait a while, then use any sharp tool (such as a sickle or dagger) to harvest the fully grown crop.
+
+  To keep your crops alive for an extended period, be sure to water them frequently. Keeping plants healthy is easier with the spell **“Plant Growth”** from the Vivification tree, which restores a plant’s water, nutrients, and health.
+
+  If all your seedbeds are full and demand is still high, you can use a hoe to till additional soil. Consider making a seed bag if you plan to expand your farm.
+
+  ## The Cellar
+  The cellar is extremely popular among patrons. Several barrels can be found here that dispense alcohol. Grab a mug, ensure you are drawing from a keg, and profit.
+
+  A typical cellar also contains a few tables and torches, as well as a barrel of water. Tables usually hold spare mugs. Be careful not to serve too much alcohol at once... unless your esteemed customer is a dwarf.
```

Comment:
The cellar is located in the caves. And ill-wishers can break the "wall" and get into it, breaking all the barrels. You need to check them periodically.
If the barrels are broken and the liquid is on the floor - it can be collected in a bucket and "mixed all in one barrel", creating a "Cocktail", which is also good for serving.